recipe/build.sh

```diff
@@ -102,9 +102,11 @@ cmake .. -LAH                                                             \
     -DWITH_FFMPEG=1                                                       \
     -DWITH_MATLAB=0                                                       \
     -DWITH_VTK=0                                                          \
+    -DWITH_QT=5                                                           \
     -DWITH_GPHOTO2=0                                                      \
     -DINSTALL_C_EXAMPLES=0                                                \
     -DOPENCV_EXTRA_MODULES_PATH="../opencv_contrib-$PKG_VERSION/modules"  \
+    -DBUILD_opencv_cvv=OFF                                                \
```

Comment:
Right, I noticed the builds failing on py2.7 + win. See: https://ci.appveyor.com/project/conda-forge/opencv-feedstock/build/1.0.204, I stopped the builds early but they were succeeding on py35. 

I decided to disable for all platforms uniformly (it wasn't enabled on any platform prior to this PR anyway), but if you think we should disable it only in that build combination, I can try that.